op-batcher/go.mod

```diff
@@ -2,12 +2,15 @@ module github.com/ethereum-optimism/optimism/op-batcher
 
 go 1.18
 
+replace github.com/ethereum-optimism/optimism/op-service v0.0.0 => ../op-service
+
 require (
 	github.com/ethereum-optimism/optimism/op-node v0.3.0
 	github.com/ethereum-optimism/optimism/op-proposer v0.3.0
+	github.com/ethereum-optimism/optimism/op-service v0.0.0
 	github.com/ethereum/go-ethereum v1.10.21
 	github.com/miguelmota/go-ethereum-hdwallet v0.1.1
-	github.com/urfave/cli v1.22.5
+	github.com/urfave/cli v1.22.9
```

Comment:
Perhaps out of scope of this PR, but upstream has migrated to `urfave/cli` v2
https://github.com/ethereum/go-ethereum/blob/c0cc6f63629678ad7a8d9a33dc17bcf26122bde0/go.mod#L58

Reply:
This'll require a much larger refactor - can take care of that separately.